app/views/govuk_publishing_components/components/docs/govspeak.yml

```diff
@@ -50,7 +50,12 @@ examples:
         </ul>
         <p>A paragraph of text following the list. I'm not going to talk about the words anymore.</p>
         <blockquote>
-          <p>This text is inside a blockquote.</p>
+          <p>This text is inside a blockquote. It is being made deliberately long so as to wrap onto a second line, which hopefully has happened by now.</p>
+        </blockquote>
+        <blockquote>
+          <p>This text is inside a blockquote with a citation.</p>
+          <p>It contains more than one paragraph, and also has extra words in order to make it wrap onto a second line so we can check that everything's okay.</p>
+          <cite>John Blockquote, inventor of the blockquote</cite>
```

Comment:
Would there be any mileage in adding this example to the [individual blockquote example](https://github.com/alphagov/govuk_publishing_components/blob/41b98c097a0142e096dfd77ce2ff079b180fc509/app/views/govuk_publishing_components/components/docs/govspeak.yml#L464) too? Only because when I was reviewing and refreshing the page I had to search a bit to find the blockquote amongst the other common elements.

This might just be me though so totally happy to leave things are they are if it's not an issue.

Reply:
Good shout, have copied and pasted it 👍